my-app/src/app/home/quick-links/quick-links.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="quick-links-grid-container">
+    <div>
```

Comment:
I want you to center these links in the center of the page, and center the text inside of them as well.